Don't attempt to use /usr/bin/wx-config-3.0 which can point to the
wrong wx-config depending on eselect'ed wxGTK profile. Furthermore
that path does not use ESYSROOT.

If unset, FindwxWidgets.cmake will use WX_CONFIG environment
set by the wxwidgets eclass.
--- a/cmake/SearchForStuff.cmake
+++ b/cmake/SearchForStuff.cmake
@@ -50,4 +50,5 @@
 # lib32-wx3.0 => /usr/bin/wx-config32-3.0
 # FindwxWidgets only searches for wx-config.
+#[[
 if(CMAKE_CROSSCOMPILING)
     # May need to fix the filenames for lib32-wx3.0.
@@ -71,4 +72,5 @@
     endif()
 endif()
+#]]
 
 find_package(wxWidgets COMPONENTS base core adv)
